Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -9,6 +9,7 @@ import Shared
 import Storage
 import SnapKit
 import XCGLogger
+import AVFoundation
```

Comment:
Doesn't look like this import is necessary since you extracted out the AV-specific code to AuralProgressBar.swift.